### ISSUE TITLE
AG Grid React > ReactUI > LoadingRenderer Bug

### ISSUE THREAD

**Issue by cssagogo** [ag-grid/ag-grid#4771]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
This issue is also described here: https://stackoverflow.com/questions/69946917/ag-grid-react-reactui-loading-bug

I'm getting an odd behavior when trying to switch my React AG Grid to use the ReactUI param as described here: https://www.ag-grid.com/react-data-grid/reactui/#js-components-inside-react-ui

When I add the following to my AgGridReact component:
```
reactUi
```
The loaders I have setup using the following don't clear after the content is loaded:
```
components={{
  loadingRenderer: (params) => {
    if (params.value !== undefined) {
      return params.value
    } else {
      return '<img src="https://www.ag-grid.com/example-assets/loading.gif">'
    }
  }
}}
```
It works as expected if I remove the reactUi param. Here is a plunker I have setup that illustrates the problem.

https://plnkr.co/edit/D2bKgj6USnIhO1Tj

Is there an alternative approach to setting up the loader or is this a bug in AG Grid?

**Expected behavior**
The loaders should be removed when the content loads.

**Please tell us about your environment:**
Testing on a Mac in Chrome.

* **AG Grid version:** 26.1.0
    "ag-grid-community": "^26.1.0",
    "ag-grid-react": "^26.1.0",
    
* **Browser:**
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.69 Safari/537.36'

* **Language:** [ES6/7 | React]

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by icedjello** [comment]
I looked into your scenario and built a sample based on your code.
https://plnkr.co/edit/hXsOIG0HvQAFZBn9 
This problem has been fixed in version 26.2.0 released today!

I hope this helps!
